src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/DocumentPresentation/AbstractTextDocumentPresentationEndpointBase.cs

```diff
@@ -198,7 +181,7 @@ private bool IsVirtualDocumentUri(Uri uri)
                 var uri = new Uri(entry.Key);
                 var edits = entry.Value;
 
-                if (!IsVirtualDocumentUri(uri))
+                if (!_languageServerFeatureOptions.IsVirtualDocumentUri(uri))
```

Comment:
At first I thought `LanguageServerFeatureOptions` was the wrong place for `IsVirtualDocumentUri` but now I'm thinking it's probably the right place and maybe the name is just no longer reflective? I usually assume an "Options" class is just a POCO.

Reply:
Yeah, the name on this side of the fence is weird. on the other side of the fence, its `RazorConventions` and I kinda like that better, but I don't know.